extension/source/types/Rpc.ts

```diff
@@ -126,13 +161,14 @@ export const rpcMap = {
   },
 
   // TransactionController
+
   createTransaction: {
-    origin: '*',
+    origin: '<quill>',
```

Comment:
I made these all `<quill>` methods since dApps shouldn't be calling them, afaict. E.g. `createTransaction` will insert a transaction into our history even if it isn't initiated, so I really don't think it should be externally callable.